### ISSUE TITLE
Console logs

### ISSUE THREAD

**Issue by muuvmuuv** [jepiqueau/jeep-sqlite#28]
Seeing again a lot of console logs. Maybe hide it behind a flag for development?

```
>>>> in runSQL: INSERT INTO key_value (key, value) VALUES (?, ?) ON CONFLICT (key) DO UPDATE SET value=excluded.value, false , false
>>>> in runSQL: meta,{"speakableLanguages":[{"id":1,"displayName":{"de-DE":"Afar","en-GB":"Afar"},"isoShort":"aa"},{"id":2,"displayName":{"de-DE":"Abchasisch","en-GB":"Abkhazian"},"isoShort":"ab"},{"id":3,"displayName":{"de-DE":"Afrikaans","en-GB":"Afrikaans"},"isoShort":"af"},{"id":4,"displayName":{"de-DE":"Akan","en-GB":"Akan"},"isoShort":"ak"},{"id":5,"displayName":{"de-DE":"Albanisch","en-GB":"Albanian"},"isoShort":"sq"},{"id":6,"displayName":{"de-DE":"Amharisch","en-GB":"Amharic"},"isoShort":"am"},{"id":7,"displayName":{"de-DE":"Arabisch","en-GB":"Arabic"},"isoShort":"ar"},{"id":8,"displayName":{"de-DE":"Aragonesisch","en-
```

> https://github.com/jepiqueau/jeep-sqlite/blob/a7566249cddabddb737281fe0ed4f6c3f7e168e8/src/utils/database.ts#L397

**Comment by muuvmuuv** [comment]
@jepiqueau I would like to work on that and do a PR. Are you fine with a simple logger class that accepts simply a "show logs in console" bool?

**Comment by muuvmuuv** [comment]
Nvm you already fixed it in 2.5.2. Closing, thanks!

**Comment by jepiqueau** [comment]
@muuvmuuv sorry i missed them when i published 2.5.1 it is fixed in 2.5.2 that i published this morning
